scrapers/Spacebattles/Spacebattles/spiders/spacebattles_scraper.py

```diff
@@ -0,0 +1,191 @@
+            story, created = Story.objects.get_or_create(title=title)
+            story.save()
+            story.authors.add(author)
+
+            a_node = thread_tag.xpath("div/div/h3/a")
+            thread_url = a_node.xpath("@href").extract_first()
+
+            # print("\nauthors: {0}".format(author.name))
+            # print(  "    url: {0}".format(thread_url))
+            # print(  "  title: {0}".format(story.title))
+
+            if thread_url is not None:
+
+                thread_link = response.urljoin(thread_url)
+                url_stories.append((thread_link, story))
+                # yield scrapy.Request(thread_link, callback=self.scan_thread)
+
+        return url_stories
+
+    def scan_thread(self, response):
+
+        """ Scan the actual thread for story and author content
+
+        This looks for any threadmarks, processes them, and
+        follows the last threadmark's "next" link.
+        If the page doesn't have any threadmarks, or there is
+        no next link, it gracefully closes. It also creates story
+        segments, and links them to a story. (django models)
+
+
+        Keyword arguments:
+        response -- the response object used to navigate the page
+
+        META arguments:
+            story_item -- the story object to associate the
+        """
+        story_item = response.meta.get("story_item")
+        print("\nscraping thread {0}\n".format(response.url))
+
+        div_tmarks = response.xpath("//li[contains(@class, 'hasThreadmark')]")
+        
+        if div_tmarks is not None and len(div_tmarks) > 0:
+
+            for div_tmark in div_tmarks:
+                story_seg = StorySegment()
+
+                author = div_tmark.xpath("@data-author").extract_first()
+
+                author_seg, created = Author.objects.get_or_create(name=author)
+                story_item.authors.add(author_seg)
+
+                title = "".join(div_tmark.xpath("div/span/text()").extract()).encode('utf-8')
+                title = " ".join(title.split())
+                story_seg.title = title
+                story_seg.story = story_item
+
+                date_time = div_tmark.xpath(".//span[@class='DateTime']/@title").extract_first()
+                if date_time is None:
+                    date_time = div_tmark.xpath(".//abbr[@class='DateTime']/text()").extract_first()
+                date_obj = datetime.strptime(date_time, "%b %d, %Y at %I:%M %p")
+                story_seg.published = date_obj
+
+                # TODO ^^ Convert date time to actual date_time type
+
+                content = div_tmark.xpath(".//blockquote/text()").extract_first().encode('utf-8')
+                content = " ".join(content.split())
+                story_seg.contents = content
+
+                print("Title: {0}   Author: {1}".format(story_seg.title, author))
+                print("date_time: {0}".format(date_obj))
+                # print("content: {0}".format(content[0:200]))
```

Comment:
For debugging, it's not always necessary to print out so much text from a story. :/